src/main/java/digital/slovensko/autogram/core/SigningParameters.java

```diff
@@ -227,4 +229,8 @@ public boolean shouldCreateDatacontainer() {
     public boolean getCheckPDFACompliance() {
         return checkPDFACompliance;
     }
+
+    public int getPreviewWidth() {
+        return (previewWidth > 0) ? previewWidth : 640;
```

Comment:
Tuto ale vyzera ze default neostal 600, toto menime hej?

Reply:
640 je hodnota z toho enumu, ktorá je najbližšie pôvodným 600. Dáva mi zmysel, že default bude niečo z enumu a nie úplne iné.